### ISSUE TITLE
Accessibility and contrast 

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-dropdown#40]
## Describe the bug

Are these colors correct? 

<img width="631" alt="Screen Shot 2021-12-02 at 4 43 38 PM" src="https://user-images.githubusercontent.com/181089/144525471-8a489578-2fe1-49c6-bcd1-b65949f15d86.png">

The accent UI elements will not pass the WCAG a11y contrast ratio. While this is not text, these are key affordances for user experience and should live up a proper contrast ratio.

**Comment by jason-capsule42** [comment]
@blackfalcon Is this a duplicate of https://github.com/AlaskaAirlines/auro-dropdown/issues/33

**Comment by blackfalcon** [comment]
Yes.
